central/compliance/service/service_impl.go

```diff
@@ -127,9 +127,19 @@ func (s *serviceImpl) GetAggregatedResults(ctx context.Context, request *v1.Comp
 		return nil, err
 	}
 
+	filteredSources := sources[:0]
+	for _, source := range sources {
+		config, exist, errConfig := s.complianceDataStore.GetConfig(ctx, source.GetStandardId())
+		if errConfig != nil {
+			return nil, errConfig
+		}
+		if !exist || !config.GetHideScanResults() {
```

Comment:
if you want to make the logic cleaner then I would suggest using the if and then continue. e.g. flipping it so that you continue on negative cases

```
if !exists || config.GetHideScanResults() {
  continue
}
filteredSources = append(filteredSources, source)
```

This increases readability and makes it obvious that the cases you are checking for disqualify the source

Reply:
Logic is correct. If standard does not exist it means it is not hidden

Reply:
ah its just the existence of the config for the standard so we should then do if exists && config.GetHideScanResults {skip}